issue: npm ERR Could not resolve dependency

### Version Number

1.6.3

### Codesandbox/Expo snack

_No response_

### Steps to reproduce

1. npm install @ts-react/form
2. npm install react-hook-form @hookform/resolvers
3. npm update


### Expected behaviour

No npm Errors.

### Relevant log output

```shell
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: bulletproof-react-vite@0.0.0
npm ERR! Found: @hookform/resolvers@3.1.0
npm ERR! node_modules/@hookform/resolvers
npm ERR!   @hookform/resolvers@"^3.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @hookform/resolvers@"^2.8.0" from @ts-react/form@1.6.3
npm ERR! node_modules/@ts-react/form
npm ERR!   @ts-react/form@"^1.6.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
     "!lib/src/__tests__"
   ],
   "peerDependencies": {
-    "@hookform/resolvers": "^2.8.0",
+    "@hookform/resolvers": "^2.8.0 || ^3.0.0",
     "react": "^16.8.0 || ^17 || ^18",
     "react-hook-form": "^7.39.0",
     "zod": "^3.19.0"

